### ISSUE TITLE
Failure to install version 0.4 with pip>=10.0.0 on Python 2.7

### ISSUE THREAD

**Issue by jaraco** [python/importlib_resources#53]
In GitLab by @Midnighter on Apr 24, 2018, 19:27

This was caught by our CI. The relevant error message is:

```
Missing build time requirements in pyproject.toml for importlib_resources from https://files.pythonhosted.org/packages/cd/d7/47a35e84a4f48aece34095f7c02da3100384fc6400a8139f88a18023f34f/importlib_resources-0.4.tar.gz#sha256=dcfa996d4f988c5a4244c5a62c3decfc01488a19afa459726eac47ed41ec0c1d (from memote==0.6.2): 'setuptools' and 'wheel'.
  This version of pip does not implement PEP 517 so it cannot build a wheel without 'setuptools' and 'wheel'.
  Could not find a version that satisfies the requirement flit (from versions: )
No matching distribution found for flit
```

Either importlib_resources==0.3 or pip<10.0.0 work just fine.

**Comment by jaraco** [comment]
In GitLab by @warsaw on May 1, 2018, 12:56

There's definitely something weird about this.  I noticed it when I was working on #52 - I don't really know what's going on but I suspect it's some incompatibility with newer pip and PEP 517.  I've moved `pyproject.toml` aside for the 0.5 release.  We'll just rely on the `setup.py` for now, but it would be good to investigate further and possibly report the problem to `pip`.
